docs/user-guide/doc-odm-user-guide/import-samples-spreadsheet.md

```diff
@@ -95,43 +121,41 @@ In the subsequent dialog window, several parameters need to be defined:
 
-- Measurement Separator Symbol in Column Name: Each column name should contain a symbol (or symbols combination), separating sample (library or preparation) name from measurement type, e.g., it would be a dot for ‘Sample1.p-value’. If a column name has more than one measurement separator (for instance, ‘Sample1.p.value’), the first one will be used for separation.
-- Measurement Separator Symbol on upload request: The separator must be explicitly specified on data upload request either through API or GUI.
-- Presence of Measurement Separator Symbol: Every column must include a measurement separator.
-- Consistency of Measurement Types: All samples (libraries or preparations) must have the same types of measurements in the file. For example, if you have three samples and each has measurements for Intensity and Quality Pass, your file should have six columns named: ‘Sample1.Intensity’, ‘Sample1.QualityPass’, ‘Sample2.Intensity’, ‘Sample2.QualityPass’, ‘Sample3.Intensity’, ‘Sample3.QualityPass’.
+![58ImportData2.png](doc-odm-user-guide/images/58ImportData2.png)
+<figcaption>Select the source for the experimental data. Experimental data can be imported from your local computer or a cloud-based storage system (such as AWS)</figcaption>
 
-In the second step, you’ll need to choose the file you wish to upload. There are two options:
+### **Linking Data**
 
-- **Local Computer**: Select the file located on your computer. Please note: this option is only available if your ODM version is coupled with AWS S3 storage (provided by default). If this option isn’t functioning properly, please get in touch with Genestack’s customer care team.
-- **External Link**: Upload the file by supplying a link to the file’s location. Keep in mind that while this option is intended to support any external location, in its BETA version, it only supports links from AWS S3 storage. If you encounter any issues with this feature, please reach out to Genestack’s customer care team.
+* **Default Linking**: By default, the data is linked with the Samples file using the **Sample Source ID** column. To ensure proper linking, make sure your file includes a column called **Sample Source ID** with the same IDs used in the Sample Metadata table uploaded previously (see section "Import Samples Metadata").
+* **Custom Linking**: Alternatively you can select a different column to link the **experimental** data, such as **Sample Name**, **Date**, etc. This provides flexibility in how data is associated, but it is recommended to include the **SampleSourceID** column for consistent referencing and linking samples metadata files with additional data types like libraries and preparations. Read the [Supported format files](https://docs.google.com/document/d/1uF3g11QO3PfyhsSY7u0CPozn0LPRsoblnI2djZydvIs/edit?usp=sharing) section for more information.
```

Comment:
Please, rename SampleSourceID to Sample Source ID, also please, replace link to some google doc with some internal link from the user guide